### PR TITLE
Contributing

### PR DESCRIPTION
**Pull request by bluebeel**
Update CONTRIBUTING.md

### CODE REVIEW

File: documentation/CONTRIBUTING.md [hunk 1]
```diff
@@ -1,2 +1,64 @@
...
+  * [Git Commit Messages](#git-commit-messages)
+  * [Python Styleguide](#python-styleguide)
+  * [C++ Styleguide](#c++-styleguide)
+  * [Documentation Styleguide](#documentation-styleguide)
+
+
+  ## How Can I Contribute?
+
+  ### Your First Code Contribution?
+  To ensure some quality control and cohesion in the code, each contribution must be reviewed and approved by a member of the ** reviewers ** team. No one has the rights to make commits directly on the deposit. This is why everyone must create a personal copy of the repo * (= a fork) * on which they can make commits. Once the change is ready to be embedded in the original code, simply create a ** Pull-Request ** so that the change can be replayed and integrated.
```
Comment (azerupi) [inline documentation/CONTRIBUTING.md]: Markdown italic and bold starts can not have spaces in between them and the words.

Comment (azerupi) [inline documentation/CONTRIBUTING.md]: "deposit" --> "master branch" ?

Comment (azerupi) [inline documentation/CONTRIBUTING.md]: "replayed" --> "reviewed" ?

File: documentation/CONTRIBUTING.md [hunk 2]
```diff
@@ -1,2 +1,64 @@
...
+  The steps to follow are as follows:
+  1. Create a **fork** of the repository.
+  ![Fork](assets/tuto-contribution-fork.png)
+
+  2. You will have a copy of the repository under your personal namespace that you will be able to clone. Make sure the repository you are looking at is the fork under your name and not the original. Then press the ** Clone or download ** button in green. Copy the link that appears.
+  ![Clone](assets/tuto-contribution-clone.png)
+
+  3. Open the terminal and type the following command, replacing `<link>` with the link you just copied:
+  ```
+  git clone <lien>
```
Comment (azerupi) [inline documentation/CONTRIBUTING.md]: "lien" --> "link"

File: documentation/CONTRIBUTING.md [hunk 3]
```diff
@@ -1,2 +1,64 @@
...
+  ```
+  4. The repository should be cloned on your computer. You can make your changes. Once finished, you must commit changes. It is advisable to always work on another branch and not directly on the master branch.
+  ```
+  git checkout -b <branch-name> // Create and swap to the branch with name: <branch-name>
```
Comment (azerupi) [inline documentation/CONTRIBUTING.md]: Comments in bash are made with a hastag `#`

**Comment by bluebeel** [comment]
Requested changes made!

**Comment by azerupi** [comment]
Thanks, looks ready for merging! :)

**Pull request merged**